03-task-manager/starter/app.js

```diff
@@ -1 +1,28 @@
-console.log('Task Manager App')
+const express = require('express')
+const app = express()
+const tasks = require('./routes/tasks')
+const connectDB = require('./db/connect')
+require('dotenv').config()
+
+//Middleware
+app.use(express.json())
+
+//ROutes
+app.get('/hi',(req,res)=>{
+    res.send('Task Manager')
+})
+
+app.use('/api/v1/tasks',tasks)
+
+const port = 3000
+
+const start = async () =>{
+    try {
+        await connectDB(process.env.MONGO_URI)
+        app.listen(port,console.log(`ecuchando en puerto ${port}`))
```

Comment:
Oh, you probably want to wrap `console.log(`ecuchando en puerto ${port}`)` in a callback function. As is, the console.log statement will be invoked immediately rather than after the app actually starts. See here for an example of how it's supposed to work: https://expressjs.com/en/starter/hello-world.html

Reply:
Ok, thank you.